src/ml/neural_net/image_augmentation.hpp

```diff
@@ -288,7 +288,7 @@ class float_array_image_augmenter : public image_augmenter {
   /** The output sent from TensorFlow after augmenting the images. */
   struct float_array_result {
     /** The images after augmenting sent from Tensorflow */
-    shared_float_array images;
+    std::vector<shared_float_array> images;
```

Comment:
This might have made the struct (labelled_float_image) that is going into tensorflow and this the same. Can we just remove one of them?

Reply:
Good call!